koku/api/report/test/ocp/tests_views.py

```diff
@@ -220,9 +218,10 @@ def test_generic_report_ocp_cpu_success(self, mock_handler):
         request = Request(django_request)
         request.user = user
 
-        extras = {'report_type': 'cpu'}
-        response = _generic_report(request, QueryParamSerializer, OCPReportQueryHandler, **extras)
+        response = _generic_report(request, report='cpu', provider='ocp')
         self.assertIsInstance(response, Response)
+        # FIXME
+        # self.assertEqual(response.status_code, 200)
```

Comment:
Concerning that it's not returning a 200. What's up here?

Reply:
Not my test. I don't know.

The test itself broke during my refactor, but was fixed by something else. So, I just left a note while I was in there that this test was missing a check of the return code. 

Currently, the test only checks that it gets a response, not whether the response is good.